docs/01-development/01-contributing.md

```diff
@@ -265,4 +269,32 @@ kind delete cluster --name=<cluster name>
 
 By default, test namespaces created with `CreateTestNamespace` are automatically deleted when a test ends. This is inconvenient for debugging failed tests.
 
-This behavior is controlled by passing the `WithKeepOnFail` option when creating the namespace or by setting `E2E_KEEPONFAIL=true` in the environment when running an e2e test.
\ No newline at end of file
+This behavior is controlled by passing the `WithKeepOnFail` option when creating the namespace or by setting `E2E_KEEPONFAIL=true` in the environment when running an e2e test.
```

Comment:
Other env vars are set to `0` or `1`, but this one we set it to `true`. IIRC both are okay as we use `strconv.ParseBool` which accepts both, but I think it would be good to stick to one or the other in the examples.

Reply:
Good catch. It should also be `1`.